src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index].name().toLowerCase();

```

Comment:
I think it isn't necessary
```suggestion
        return Color.values()[index].name();

```